Engine/Rendering/LightFrustum.cpp

```diff
@@ -10,6 +10,8 @@
 #include "Module/ModuleSpacePartitioning.h"
 #include "Module/ModuleRender.h"
 
+const int LIGHT_FRUSTUM_FAR_PLANE_EXTRA_DISTANCE = 10.f;
```

Comment:
put this inside a namespace, is good practice because this way you make sure you are not going to have issues if another LIGHT_FRUSTUM_FAR_PLANE_EXTRA_DISTANCE  const is defined in some include header.

Reply:
Will do!